### ISSUE TITLE
GXLE cannot be installed because of the X-Score inaccessibility

### ISSUE THREAD

**Issue by mglgc** [LinaDongXMU/GXLE#1]
Hi,
First of all, thank you for freely making available to the scientific community the GXLE code. We are trying to install and testing it but with unsuccessful results because there is no way to download the X-Score from https://www.ics.uci.edu/~dock/manuals/xscore_1.2_manual.
Please, could you provide an alternative way to get such a requirement, e.g. including X-Score into the GXLE conda environmente or whatever any other fix you decide? 
Thank you very much in advance for your reply.

**Comment by LinaDongXMU** [comment]
Hi,
Thank you for following. The current download address for X-Score is here (http://www.sioc-ccbg.ac.cn/?p=42), and registration is required at first. I hope it will be helpful to you.

**Comment by mglgc** [comment]
> Hi, Thank you for following. The current download address for X-Score is here (http://www.sioc-ccbg.ac.cn/?p=42), and registration is required at first. I hope it will be helpful to you.

Dear LinaDongXMU, thank you very much for the working link. Regards.
